tests/integration/test_kinesis.py

```diff
@@ -89,6 +90,9 @@ def test_subscribe_to_shard(self):
         results = []
         for entry in stream:
             records = entry['SubscribeToShardEvent']['Records']
+            continuation_sequence_number = entry['SubscribeToShardEvent']['ContinuationSequenceNumber']
+            # https://docs.aws.amazon.com/kinesis/latest/APIReference/API_SubscribeToShardEvent.html
+            self.assertTrue(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number) is not None)
```

Comment:
nitpick: could be changed to either:
```
self.assertIsNotNone(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number))
```
or:
```
self.assertTrue(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number))
```

Reply:
Sweet, I'll change this. First time I've touched python in quite a while :)